app/room/[roomId]/page.js

```
@@ -0,0 +1,327 @@
+          </div>
+          
+          {/* Remote video */}
+          {isConnected && (
+            <div className="video-container bg-black rounded-lg overflow-hidden relative aspect-video">
+              <video 
+                ref={remoteVideoRef}
+                className="w-full h-full object-cover"
+                autoPlay 
+                playsInline
+              />
+              <div className="absolute bottom-2 left-2 bg-black/50 text-white px-2 py-1 rounded text-xs">Remote User</div>
```

Biome (1.9.4)

[error] Provide a track for captions when using audio or video elements.

Captions support users with hearing-impairments. They should be a transcription or translation of the dialogue, sound effects, musical cues, and other relevant audio information.

(lint/a11y/useMediaCaption)